packages/components/tests/styles-test.js

```diff
@@ -22,7 +22,7 @@ const files = glob.sync('**/*.scss', {
 const render = promisify(sass.render);
 
 describe('styles', () => {
-  jest.setTimeout(8000);
+  jest.setTimeout(10000);
```

Comment:
FWIW I had to change this to `12000` to not get an error 🤷‍♂

Reply:
Ooof, same. Let me bump up to 20000 just in case